svthalia/concrexit
# Remove image rotation support

Closes #3489.



### Summary

Remove photo.rotation. Also check in the migration that there are no rotated photos to prevent data loss.

### How to test

1. Check that it fails when there are rotated images
2. Check that it works otherwise
3. Grep for remaining references?
4. ???
